src/internal_modules/roc_core/target_posix/roc_core/thread.cpp

```diff
@@ -57,6 +57,7 @@ bool Thread::enable_realtime() {
                 errno_to_str(err).c_str());
         return false;
     }
+    roc_log(LogDebug, "thread: set realtime priority");
```

Comment:
Let's move this log before actually calling pthread_setschedparam, so that error will go after, like this:

```
    roc_log(LogDebug, "thread: setting realtime priority");

    if (int err = pthread_setschedparam(pthread_self(), SCHED_RR, &param)) {
        roc_log(LogError,
                "thread: can't set realtime priority: pthread_setschedparam(): %s",
                errno_to_str(err).c_str());
        return false;
    }
```

Also let's use LogError instead of LogDebug in the error branch.